pkg/oc/admin/prune/images.go

```diff
@@ -145,7 +145,7 @@ func NewCmdPruneImages(f *clientcmd.Factory, parentName, name string, out io.Wri
 	}
 
 	cmd.Flags().BoolVar(&opts.Confirm, "confirm", opts.Confirm, "If true, specify that image pruning should proceed. Defaults to false, displaying what would be deleted but not actually deleting anything. Requires a valid route to the integrated Docker registry (see --registry-url).")
-	cmd.Flags().BoolVar(opts.AllImages, "all", *opts.AllImages, "Include images that were not pushed to the registry but have been mirrored by pullthrough.")
+	cmd.Flags().BoolVar(opts.AllImages, "all", *opts.AllImages, "Include images are imported from external registries.")
```

Comment:
I think it'd read a bit better with `that` after `Include`: `Include images that are...`

Reply:
err, yeah. totally left a word out. thanks @juanvallejo 

updated.